mmv1/third_party/terraform/services/cloudquotas/resource_cloud_quotas_quota_preference_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Nit: license added twice. You actually shouldn't need it at all in this repository.